File: Web.ClientApp/src/shared/contracts/player-info-response.ts
export interface IPlayerInfoResponse {
    id: string;
    avatarUrl: string;
    playerName: string;
    matches: number;
    loses: number;
    wins: number;
    draws: number;
    allExp: number;
    currentExp: number;
    targetExp: number;
    money: number;
    likes: number;
    level: number;
    has21: number;
}

export type PlayerInfo = IPlayerInfoResponse;

File: src/Services/App/App.Contracts/Identity/Responses/PlayerInfoResponse.cs
namespace App.Contracts.Identity.Responses;

public sealed record PlayerInfoResponse(
    Guid Id,
    string AvatarUrl,
    string PlayerName,
    int Matches,
    int Loses,
    int Wins,
    int Draws,
    int AllExp,
    int CurrentExp,
    int TargetExp,
    int Money,
    int Likes,
    int Level,
    int Has21);